pages/[report]/[fight]/index.tsx

```diff
@@ -1,118 +1,94 @@
 }
 
-const Fight = ({ fight, report }: IFight) => {
-  console.log('loggi ', fight);
+const Fight = ({ singleReport, playerDetails }: IDisplayFight) => {
   const router = useRouter();
+  const selectPlayer = (id: number) => {
+    router.query.playerID = id.toString();
+    router.push(router);
+  };
+
+  useEffect(() => {
+    const playerContainer = document.getElementById('playerDetails');
+    console.log(playerContainer);
+    if (router.query.playerID !== undefined && playerContainer !== null) {
+      playerContainer.scrollIntoView({
+        behavior: 'smooth',
+      });
+    }
+  }, [router.query.playerID]);
 
-  function UpdateQueryParam({ id, value }: IUpdateQueryParam) {
-    router.query[id] = value.toString();
-    router.push({
-      query: { ...router.query },
-    });
-  }
   return (
     <Main>
-      <EncounterContainer>
-        <p>
-          Boss:
-          {fight.name}
-        </p>
-        <p>
-          From:
-          {' '}
-          {dateFormat(report.startTime, 'mmmm dS, yyyy, h:MM TT')}
-        </p>
-        <p>
-          Till:
-          {dateFormat(report.endTime, 'mmmm dS, yyyy, h:MM TT')}
-        </p>
-        <p>
-          Faction:
-          {report.guild.faction.name}
-        </p>
-        <p>
-          Guild:
-          {report.guild.name}
-        </p>
-      </EncounterContainer>
-      <CharacterContainer>
-        {fight.friendlyPlayers.map((player) => (
-          <CharacterInfo key={player} onClick={() => UpdateQueryParam({ id: 'player', value: player })}>
-            Name:
-            {' '}
-            {player}
-          </CharacterInfo>
-        ))}
-      </CharacterContainer>
+      <NavBar>
+        <LogMetaData {...singleReport} />
+        {singleReport.fights
+          && singleReport.fights.map((fight) => <FightMetaData key={fight.id} {...fight} />)}
+      </NavBar>
+      <Content>
+        <PlayerTypeList playerDetails={playerDetails} selectPlayer={selectPlayer} />
+        <FeatureFilter />
+      </Content>
+      {router.query?.playerID && (
+      <PlayerDetail id="playerDetails">
+        h2
+      </PlayerDetail>
+      )}
     </Main>
   );
 };
 
 export default Fight;
-Fight.getInitialProps = async ({ query }: NextPageContext) => {
-  const { fight, report } = query;
-  try {
-    console.log(report);
-    // const res = await swr.get(`${process.env.BACKEND_URL}`);
-    // const data = res.data;
-    return {
-      report: GetBossFightsToLog.data.reportData.report,
-      fight: GetBossFightsToLog.data.reportData.report
-        .fights
-        .find((curFight) => curFight.id.toString() === fight),
-    };
-  } catch (error) {
-    return { error };
-  }
+
+export const getServerSideProps: GetServerSideProps = async (props) => {
+  const { query } = props;
+  console.log('process: ', `${process.env.BACKEND_URL}?action=fight&code=${query?.report}&fight=${query?.fight}&encounterID=${query?.encounterID}`);
+  const { data } = await axios.get(
```

Comment:
`const { data } = await axios.get(` could also be `const { { singleReport, playerDetails } } = await axios.get(`